lib/surface/compiler/tokenizer.ex

```diff
@@ -111,6 +111,16 @@ defmodule Surface.Compiler.Tokenizer do
     handle_macro_body(rest, line + 1, state.column_offset, ["\n" | buffer], acc, state)
   end
 
+  defp handle_macro_body("</#raw" <> rest, line, column, buffer, acc, state) do
+    handle_tag_close(
+      "#raw" <> rest,
+      line,
+      column + 2,
+      text_to_acc(buffer, acc),
+      state
+    )
+  end
+
   defp handle_macro_body("</#" <> <<first, rest::binary>>, line, column, buffer, acc, state)
        when first in ?A..?Z do
```

Comment:
@lnr0626 It was my first implementation but it does not work because `name` is not equal to the content of `<< >>` so it means it contains everything before (`</#`) and after. And we do not know what comes after, for example `rest` could contain `\n` or not. This is why I added the following test: https://github.com/surface-ui/surface/blob/7a31ad2c9eba4abbefd3243724d0d5386ba26b5b/test/compiler/tokenizer_test.exs#L644-L657

Reply:
ahhh, that makes sense